src/Parser/CMakeLists.txt

```diff
@@ -1,10 +1,13 @@
 set (INCLUDE_ROOT ${PROJECT_SOURCE_DIR}/src/Parser)
 set (SRC_ROOT ${PROJECT_SOURCE_DIR}/src/Parser)
+set (ARG_ROOT ${SRC_ROOT}/ArgumentsParser)
 
 set (SRC
     ${SRC_ROOT}/IParser.hpp
     ${SRC_ROOT}/AParser.hpp
     ${SRC_ROOT}/AParser.cpp
+    ${ARG_ROOT}/ArgumentsParser.hpp
+    ${ARG_ROOT}/ArgumentsParser.cpp
 )
 
```

Comment:
Don't forget to subdirectory to the target include directory using:
`target_include_directory(${PROJECT_NAME}_parser PUBLIC ${ARG_ROOT})`